index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const webidlWrapper = require('./webidl2js-wrapper.js');
+const webidlConversions = require('webidl-conversions');
+
+const sharedGlobalObject = {};
+webidlWrapper.install(sharedGlobalObject);
+
+const origCSSStyleDeclaration = sharedGlobalObject.CSSStyleDeclaration;
+
+/**
+ * @constructor
+ * @param {((cssText: string) => void) | null} [onChangeCallback]
+ * The callback that is invoked whenever a property changes.
+ */
+function CSSStyleDeclaration(onChangeCallback = null) {
+  if (new.target === undefined) {
+    throw new TypeError('Illegal invocation');
```

Comment:
Let's give a more useful message.

```suggestion
    throw new TypeError('Class constructor CSSStyleDeclaration cannot be invoked without 'new'');
```